mne/viz/topomap.py

```diff
@@ -468,6 +474,35 @@ def plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
     from matplotlib.widgets import RectangleSelector
 
     data = np.asarray(data)
+
+    if isinstance(pos, Info):  # infer pos from Info object
+        picks = _pick_data_channels(pos)  # pick only data channels
+        pick_info(pos.copy(), picks)
```

Comment:
...and actually I don't think this will do what you think, since it doesn't operate inplace, this will currently have no effect